Test/TestGui/TestGui.gd

```diff
@@ -12,11 +12,11 @@ func _ready():
 	Rakugo.connect("ask", self, "_on_ask")
 	Rakugo.connect("menu", self, "_on_menu")
 	Rakugo.connect("notify", self, "_on_notify")
-	
+
```

Comment:
Changes like this are due to a setting on my editor to remove whitespace form the end of editor lines.